Mathlib/RingTheory/Localization/Integral.lean

```diff
@@ -346,7 +346,7 @@ namespace IsFractionRing
 variable (R S K)
 
 /-- `S` is algebraic over `R` iff a fraction ring of `S` is algebraic over `R` -/
-theorem isAlgebraic_iff' [Field K] [IsDomain R] [IsDomain S] [Algebra R K] [Algebra S K]
+theorem isAlgebraic_iff' [Field K] [IsDomain R] [Nontrivial S] [Algebra R K] [Algebra S K]
```

Comment:
If `S` is a `Subsingleton`, then `K` would be a `Subsingleton`, which is impossible, since it's a `Field`.